src/Module/Api/Mastodon/Statuses/Bookmark.php

```diff
@@ -65,6 +65,9 @@ protected function post(array $request = [])
 
 		Item::update(['starred' => true], ['id' => $item['id']]);
 
-		System::jsonExit(DI::mstdnStatus()->createFromUriId($this->parameters['id'], $uid, self::appSupportsQuotes())->toArray());
+		// @TODO Remove once mstdnStatus()->createFromUriId is fixed
```

Comment:
What is this change supposed to do?

Reply:
It is addressing the problem I identified here: https://github.com/friendica/friendica/pull/13175#issuecomment-1591079883